2022 - Legiones Astartes.cat

```diff
@@ -46800,7 +46803,7 @@ Additionally, at the end of any Phase in which a model with Gorgon Terminator ar
             <conditionGroup type="and">
               <conditions>
                 <condition type="equalTo" value="0" field="selections" scope="force" childId="ae4a-f95c-968e-eb46" shared="true" includeChildSelections="true"/>
-                <condition type="equalTo" value="1" field="selections" scope="parent" childId="21c3-2f28-7820-e51a" shared="true" percentValue="false" includeChildSelections="true" includeChildForces="false"/>
+                <condition type="equalTo" value="1" field="selections" scope="force" childId="21c3-2f28-7820-e51a" shared="true" percentValue="false" includeChildSelections="true" includeChildForces="false"/>
```

Comment:
This is good, though also there are a few duplicates of this condition. I can clean them up.